rosdep/base.yaml

```diff
@@ -4328,6 +4332,7 @@ liborocos-kdl:
     '*': [liborocos-kdl1.4]
     bionic: [liborocos-kdl1.3]
     xenial: [liborocos-kdl1.3]
+    jammy: [liborocos-kdl1.5]
```

Comment:
Kinetic is also targeting 1.5. We should move the wildcard key to 1.5 and list out the old 1.4 versions explicitly instead of just stating Jammy so that we can keep using the wildcard hopefully in future releases.

Reply:
done